### DISCUSSION TITLE
Windows 11

### DISCUSSION THREAD

**Discussion by magicianjarden** [LizardByte/community#98] [Sunshine]
I am currently trying to run and install the Windows.exe for v0.15.0 to work. Installation completes the installation yet none of the commands that tries to run when installing and after will run. Any suggestion to what I am doing wrong?

**Comment by ReenigneArcher** [comment]
Migrating discussions to a new community repo.

I'll close this as outdated. Please let us know if you're still having issues with v0.23.1 or newer.
